Mxe Cross Bug 1

Gentle People:

    What is the best what is the best way to proceed here?

libtool: Version mismatch error.  This is libtool 2.4.6, but the
libtool: definition of this LT_INIT comes from libtool 2.4.2.
libtool: You should recreate aclocal.m4 with macros from libtool 2.4.6

    Should I down grade to version libtool 2.4.2 or
    should I recreate aclocal.m4 with macros from libtool 2.4.6?

    How do I recreate aclocal.m4?

Thomas Dineen

Failed to build package a52dec for target i686-w64-mingw32.static!
------------------------------------------------------------
Makefile:377: recipe for target 'all-recursive' failed
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory '/home/tdineen/mxe_082015/mxe/tmp-a52dec-i686-w64-mingw32.static/a52dec-0.7.4'
Makefile:412: recipe for target 'build-only-a52dec_i686-w64-mingw32.static' failed
make[1]: *** [build-only-a52dec_i686-w64-mingw32.static] Error 2
make[1]: Leaving directory '/home/tdineen/mxe_082015/mxe'
real	1m36.218s
user	0m36.164s
sys	0m32.016s
------------------------------------------------------------
[log]      /home/tdineen/mxe_082015/mxe/log/a52dec_i686-w64-mingw32.static

Makefile:412: recipe for target '/home/tdineen/mxe_082015/mxe/usr/i686-w64-mingw32.static/installed/a52dec' failed
make: *** [/home/tdineen/mxe_082015/mxe/usr/i686-w64-mingw32.static/installed/a52dec] Error 1



make[3]: Entering directory '/home/tdineen/mxe_082015/mxe/tmp-a52dec-i686-w64-mingw32.static/a52dec-0.7.4/liba52'
/bin/bash ../libtool  --tag=CC   --mode=compile i686-w64-mingw32.static-gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I../include    -std=gnu89 -O3 -fomit-frame-pointer -mcpu=pentiumpro  -prefer-non-pic -MT bitstream.lo -MD -MP -MF .deps/bitstream.Tpo -c -o bitstream.lo bitstream.c
libtool: Version mismatch error.  This is libtool 2.4.6, but the
libtool: definition of this LT_INIT comes from libtool 2.4.2.
libtool: You should recreate aclocal.m4 with macros from libtool 2.4.6
libtool: and run autoconf again.
Makefile:421: recipe for target 'bitstream.lo' failed
make[3]: *** [bitstream.lo] Error 63
make[3]: Leaving directory '/home/tdineen/mxe_082015/mxe/tmp-a52dec-i686-w64-mingw32.static/a52dec-0.7.4/liba52'
Makefile:377: recipe for target 'all-recursive' failed
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory '/home/tdineen/mxe_082015/mxe/tmp-a52dec-i686-w64-mingw32.static/a52dec-0.7.4'
Makefile:412: recipe for target 'build-only-a52dec_i686-w64-mingw32.static' failed
make[1]: *** [build-only-a52dec_i686-w64-mingw32.static] Error 2
make[1]: Leaving directory '/home/tdineen/mxe_082015/mxe'

real    1m36.218s
user    0m36.164s
sys     0m32.016s

--- a/src/a52dec.mk
+++ b/src/a52dec.mk
@@ -17,7 +17,7 @@ define $(PKG)_UPDATE
 endef
 
 define $(PKG)_BUILD
-    cd '$(1)' && autoreconf -fi # The autotools files came with a52dec are _ancient_
+    cd '$(1)' && rm aclocal.m4 && autoreconf -fi # The autotools files came with a52dec are _ancient_
     cd '$(1)' && ./configure CFLAGS=-std=gnu89 \
         $(MXE_CONFIGURE_OPTS)
     $(MAKE) -C '$(1)' -j '$(JOBS)' bin_PROGRAMS= sbin_PROGRAMS= noinst_PROGRAMS=

